host/xtest/symm_cipher_perf.c

```diff
@@ -587,18 +648,30 @@ int aes_perf_runner_cmd_parser(int argc, char *argv[])
 		} else if (!strcmp(argv[i], "-l")) {
 			NEXT_ARG(i);
 			l = atoi(argv[i]);
+		} else if (!strcmp(argv[i], "-c")) {
+			NEXT_ARG(i);
+			if (!strcasecmp(argv[i], "SM4"))
+				cipher = SM4;
+			else if (!strcasecmp(argv[i], "AES"))
+				cipher = AES;
+			else {
+				fprintf(stderr, "%s %s, invalid cipher\n",
+					xtest_progname, argv[0]);
+				USAGE();
+				return 1;
+			}
 		} else if (!strcmp(argv[i], "-m")) {
 			NEXT_ARG(i);
 			if (!strcasecmp(argv[i], "ECB"))
-				mode = TA_AES_ECB;
+				mode = ECB;
```

Comment:
s/mode/algo/

Reply:
mode is ECB, algo is TA_AES_ECB

Reply:
![image](https://github.com/OP-TEE/optee_test/assets/118411745/af77b0ef-9fdd-4584-a773-7728609e2999)

Reply:
Ok, i was wrong. It is the 4 below lines that should set `mode`, not `algo`.

Reply:
solved